toolbackup/src/main/java/io/dockstore/toolbackup/client/cli/ReportGenerator.java

```diff
@@ -4,68 +4,56 @@
 import com.google.common.reflect.TypeToken;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import org.apache.commons.io.FileUtils;
-import org.jsoup.Jsoup;
-import org.jsoup.nodes.Document;
-
 import java.io.File;
 import java.io.IOException;
 import java.lang.reflect.Type;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.Comparator;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.function.DoubleFunction;
+import org.apache.commons.io.FileUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
 
 /**
  * Created by kcao on 11/01/17.
  */
-class ReportGenerator {
+final class ReportGenerator {
+
     private static final double NUM_B_IN_GB = 1e9;
     private static final double DOUBLE_SPECIAL_NUM = 100.0;
     private static final String STYLE;
+
     static {
-        StringBuilder htmlBase = new StringBuilder("<!DOCTYPE html><html>");
-        htmlBase.append("<head>");
-        htmlBase.append("<style type=\"text/css\">\n");
-        htmlBase.append("\t\t.err { color: #ff0000; }\n");
-        htmlBase.append("\t\ttable { font-family: arial, sans-serif; border-collapse: collapse; width: 100%; }\n");
-        htmlBase.append("\t\ttd, th { border: 1px solid #dddddd; text-align: left; padding: 8px; }\n");
-        htmlBase.append("\t</style><title></title></head>");
-        htmlBase.append("<body><table>");
-
-        STYLE = htmlBase.toString();
+
+        STYLE = """
+            <!DOCTYPE html><html><head><style type="text/css">
+            \t\t.err { color: #ff0000; }
+            \t\ttable { font-family: arial, sans-serif; border-collapse: collapse; width: 100%; }
+            \t\ttd, th { border: 1px solid #dddddd; text-align: left; padding: 8px; }
+            \t</style><title></title></head><body><table>""";
     }
 
+    private ReportGenerator() {
+        // hidden constructor
+    }
+
+
+
     private static double bToGB(long sizeInB) {
-        DoubleFunction<Double> bytesToGB = (bytes) -> bytes/NUM_B_IN_GB;
-        DoubleFunction<Double> format2DecPlaces = (num) -> Math.round(num*DOUBLE_SPECIAL_NUM)/DOUBLE_SPECIAL_NUM;
+        DoubleFunction<Double> bytesToGB = (bytes) -> bytes / NUM_B_IN_GB;
+        DoubleFunction<Double> format2DecPlaces = (num) -> Math.round(num * DOUBLE_SPECIAL_NUM) / DOUBLE_SPECIAL_NUM;
```

Comment:
No need to change, but for the sake of noting, this will produce a double value that's very close to the true "rounded to hundredths" value, but most of the true values can't be represented exactly in IEEE floating point, so if care is not used when the result is later printed, something like `1.3299999999999999998` may emerge.